apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -22,10 +22,41 @@ const SameStateSwitchTemplate: StoryFn<typeof Switch> = () => {
     </div>
   );
 };
+const SwitchLabelIdTemplate: StoryFn<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  return (
+    <>
+      <label htmlFor="my-switch">My switch</label>
+      <div>
+        <Switch state={state} onChange={setState} Id="my-switch" />
+      </div>
+    </>
+  );
+};
+const SwitchNestedLabelTemplate: StoryFn<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  return (
+    <>
```

Comment:
This `<>` shouldn't be needed either, since label is the only child.

Reply:
👍 Yup fixed it. Didn't remember to question my choice here after I got this story working as intended.